### ISSUE TITLE
final fantasy explorers editor

### ISSUE THREAD

**Issue by PleaseNoSensei** [marcrobledo/savegame-editors#127]
so i figured out the file type issue but my issue now after saving the edit it converted from the standard file type for game0 but to game0.txt so how to i change the .txt to the basic game0 file

**Comment by sharrenskunk** [comment]
I'm wondering the same thing. Did you happen to figure it out?

**Comment by sharrenskunk** [comment]
Nvm. On PC it saves as .txt but on phone it doesn't modify the save and keeps it as game0

**Comment by PleaseNoSensei** [comment]
I figured it out fortunately

**Comment by PleaseNoSensei** [comment]
Yeah I had to download JkSM on my ds back up the save and move it to the pc and work from there then back on the sd can upload it to the game and bam advantage
